src/teams.rs

```diff
@@ -44,6 +53,21 @@ impl RfcbotConfig {
     pub fn should_ffcp_auto_postpone(&self, repo: &str) -> bool {
         self.fcp_behaviors.get(repo).map(|fcp| fcp.postpone).unwrap_or_default()
     }
+
+    // Update the list of teams from external sources, if needed
+    fn update(&mut self) -> Result<(), DashError> {
+        #[derive(Deserialize)]
+        struct ToDeserialize {
+            teams: BTreeMap<TeamLabel, Team>,
+        }
+        if let RfcbotTeams::Remote { ref url } = &self.teams {
+            let de: ToDeserialize = ::reqwest::get(url)?
+                .error_for_status()?
+                .json()?;
+            self.cached_teams = de.teams;
```

Comment:
Note to self: https://github.com/rust-lang/rfcs/pull/2522 would have been handy here.